diff --git a/DebugVisualizers/BUILD.gn b/DebugVisualizers/BUILD.gn
index 4e22556..ffde049 100644
--- a/DebugVisualizers/BUILD.gn
+++ b/DebugVisualizers/BUILD.gn
@@ -20,17 +20,23 @@ assert(is_win)
 config("absl") {
   # absl.natvis listed as an input in //third_party/abseil-cpp/BUILD.gn to
   # guarantee relinking on changes.
-  ldflags = [ "/NATVIS:" + rebase_path("absl.natvis", root_build_dir) ]
+  if (is_msvc) {
+    ldflags = [ "/NATVIS:" + rebase_path("absl.natvis", root_build_dir) ]
+  }
 }
 
 config("chrome") {
   # chrome.natvis listed as an input in //base:base to guarantee relinking on
   # changes.
-  ldflags = [ "/NATVIS:" + rebase_path("chrome.natvis", root_build_dir) ]
+  if (is_msvc) {
+    ldflags = [ "/NATVIS:" + rebase_path("chrome.natvis", root_build_dir) ]
+  }
 }
 
 config("blink") {
   # blink.natvis listed as an input in
   # //third_party/blink/renderer/platform/wtf to guarantee relinking on changes.
-  ldflags = [ "/NATVIS:" + rebase_path("blink.natvis", root_build_dir) ]
+  if (is_msvc) {
+    ldflags = [ "/NATVIS:" + rebase_path("blink.natvis", root_build_dir) ]
+  }
 }
