diff --git a/BUILD.gn b/BUILD.gn
index d479a4d2..425700ca 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -7366,11 +7366,15 @@ v8_component("v8_libbase") {
 
     defines += [ "_CRT_RAND_S" ]  # for rand_s()
 
-    libs = [
-      "dbghelp.lib",
-      "winmm.lib",
-      "ws2_32.lib",
-    ]
+    if (is_mingw) {
+      libs = [ "dbghelp", "winmm", "ws2_32" ]
+    } else {
+      libs = [
+        "dbghelp.lib",
+        "winmm.lib",
+        "ws2_32.lib",
+      ]
+    }
 
     if (v8_enable_etw_stack_walking) {
       libs += [ "advapi32.lib" ]  # Needed for TraceLoggingProvider.h
